docs/oxAuth_swagger.yaml

```diff
@@ -0,0 +1,2030 @@
+# raw swagger spec link:
+# https://raw.githubusercontent.com/GluuFederation/oxAuth/version_4.0/oxAuth-server/src/main/resources/swagger.yaml
```

Comment:
https://raw.githubusercontent.com/GluuFederation/oxAuth/master/docs/oxAuth_swagger.yaml

Reply:
Rectified the link